lambda/src/main/resources/db/migration/V72__TitlePublic_to_TitleClosed.sql

```diff
@@ -0,0 +1,22 @@
+-- Change name TitlePublic to TitleClosed in FileProperty and FilePropertyValues tables
```

Comment:
The migration version number in the file name will need to be updated. The latest number is `V75`. So this will need to be changed, based on the other migration PRs.